src/View/View.php

```diff
@@ -703,12 +695,12 @@ public function render($view = null, $layout = null): string
             $this->layout = $layout;
         }
 
-        $viewFileName = $view !== false ? $this->_getViewFileName($view) : null;
-        if ($viewFileName) {
+        $templateFileName = $template !== false ? $this->_getTemplateFileName($template) : null;
```

Comment:
We should document this protected method change. It will impact a few plugins.

Reply:
Hmm.. The migration guide will get bloated if we start documenting changes to protected props/methods too. Once they get a `method not found` error it shouldn't be too difficult for anyone to figure out the new method name to use :slightly_smiling_face:.

Reply:
Ok. We can always add it later if it becomes an issue.